backend/src/worker.rs

```diff
@@ -240,7 +241,7 @@ pub async fn run_worker(
                             }
                         }
                     }
-                    tracing::error!(job_id = %job.id, "Error handling job: {err}");
+                    tracing::error!(job_id = %job.id, err = err.alt(), "error handling job");
```

Comment:
The errors are also displayed in the frontend, need to check what impact this has

Reply:
`tracing::error!` shouldn't end up anywhere near the frontend/user?